.github/workflows/ci.yml

```diff
@@ -81,3 +82,21 @@ jobs:
             arduino-cli compile --fqbn esp32:esp32:esp32 /github/home/Arduino/libraries/micro_ros_arduino/examples/micro-ros_publisher -v
             arduino-cli compile --fqbn esp32:esp32:esp32 /github/home/Arduino/libraries/micro_ros_arduino/examples/micro-ros_publisher_wifi -v
             arduino-cli compile --fqbn Seeeduino:samd:seeed_wio_terminal /github/home/Arduino/libraries/micro_ros_arduino/examples/micro-ros_publisher_wifi -v
+            # BUILDING micro-ros_publisher-wifi for WiFi non-native board with ESP-AT(expected to succeed)
+            arduino-cli compile --build-property "build.extra_flags=-DUSE_WIFI_ESP_AT -DESP_AT_SERIAL_PORT=Serial1 -DESP_AT_BAUDRATE=115200 -DESP_AT_RESET_PIN=-1" --fqbn arduino:mbed:pico /github/home/Arduino/libraries/micro_ros_arduino/examples/micro-ros_publisher_wifi -v
+            # BUILDING micro-ros_publisher-wifi for WiFi native board with ESP-AT(expected to fail)
```

Comment:
Why do you expect this to fail?

Reply:
I intended to add build test that checks whether #error macro in skecth(micro-ros_publisher_wifi) works.

``` cpp
#if defined(BOARD_HAS_NATIVE_WIFI) && defined(BOARD_WITH_ESP_AT)
#error USE_WIFI_ESP_AT is not supported on boards that have native WiFi.
#endif
```

If this #error macro does not work, it will cause compilation error like below.

``` cpp
In file included from /home/administrator/Arduino/micro_ros_publisher_wifi/micro_ros_publisher_wifi.ino:21:0:
/home/administrator/Arduino/libraries/micro_ros_arduino/src/micro_ros_arduino.h: In function 'void set_microros_wifi_transports(char*, char*, char*, uint)':
/home/administrator/Arduino/libraries/micro_ros_arduino/src/micro_ros_arduino.h:134:7: error: 'static void WiFiClass::init()' is private within this context
  WiFi.init(ESP_AT_SERIAL_PORT, ESP_AT_RESET_PIN);
       ^~~~
In file included from /home/administrator/Arduino/libraries/WiFiNINA/src/WiFiNINA.h:23:0,
                 from /home/administrator/Arduino/libraries/micro_ros_arduino/src/micro_ros_arduino.h:107,
                 from /home/administrator/Arduino/micro_ros_publisher_wifi/micro_ros_publisher_wifi.ino:21:
/home/administrator/Arduino/libraries/WiFiNINA/src/WiFi.h:46:17: note: declared private here
     static void init();
                 ^~~~
In file included from /home/administrator/Arduino/micro_ros_publisher_wifi/micro_ros_publisher_wifi.ino:21:0:
/home/administrator/Arduino/libraries/micro_ros_arduino/src/micro_ros_arduino.h:134:48: error: no matching function for call to 'WiFiClass::init(arduino::UART&, int)'
  WiFi.init(ESP_AT_SERIAL_PORT, ESP_AT_RESET_PIN);
```